pydantic_ai_slim/pydantic_ai/usage.py

```diff
@@ -1,67 +1,62 @@
 from __future__ import annotations as _annotations
 
+import dataclasses
 from copy import copy
-from dataclasses import dataclass
+from dataclasses import dataclass, fields
+
+from typing_extensions import deprecated, overload
 
 from . import _utils
 from .exceptions import UsageLimitExceeded
 
-__all__ = 'Usage', 'UsageLimits'
+__all__ = 'RequestUsage', 'RunUsage', 'UsageLimits'
```

Comment:
Are we supposed to remove deprecated classes from `__all__`? I think that's a breaking change.